docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -71,9 +71,15 @@ If everything is good, you're in the right path to use {ch}!
 
 == How to setup your local cluster using minikube (if not done yet)
 
+
 {ch} is designed to help you generate all the required artifacts for a deployment on a Kubernetes cluster.
 For a local deployment, you can use minikube, which is basically https://minikube.sigs.k8s.io/docs/start/[a small {kub} cluster on your machine].
 
+
+[IMPORTANT - WSL2]
+If following this tutorial in a Windows WSL2 - then check https://stackoverflowteams.com/c/metacell/questions/84/[this] for help!
```

Comment:
It's better not to point to the stackoverflow page as it's not visible to the public. CloudHarness is open source instead. Can create a specific readme here and link to that

Reply:
@gidili suggestion sounds good. I'll do that!

Reply:
@D-GopalKrishna I was joking, no need to do that 😆